src/builder/create_sticker.rs

```diff
@@ -84,7 +84,10 @@ impl<'a> CreateSticker<'a> {
 
 #[cfg(feature = "http")]
 #[async_trait::async_trait]
-impl<'a> Builder for CreateSticker<'a> {
+impl<'a, 'de> Builder<'de> for CreateSticker<'a>
+where
+    'de: 'a,
+{
```

Comment:
I'd wonder the most about this one, I had to add this `where` clause for it to compile.

Reply:
It worked fine with just `impl<'a> Builder<'a> for CreateSticker<'a>` for me

Reply:
Oh you introduced an extra lifetime in all Builder impls? Reuse the same lifetime; adding an extra one is redundant.

Reply:
This is actually a bug - for some reason `CreateSticker::audit_log_reason` isn't marked `#[serde(skip)]`.